src/main/java/blang/distributions/F.bl

```diff
@@ -0,0 +1,41 @@
+package blang.distributions
+
+/** The F-distribution. Also known as Fisher-Snedecor distribution. */
+model F {
+  random RealVar realization
+	
+  /** The degrees of freedom \(\d_1 \) and \(\d_2 \) . \( \d_1, \d_2 > 0 \) */
+  param RealVar d1, d2
+  
+  laws {
+	logf(d1, d2) {
+		if (d1 <= 0.0) return NEGATIVE_INFINITY
+		if (d2 <= 0.0) return NEGATIVE_INFINITY
+		return 0.5 * (d1*log(d1) + d2*log(d2))
+	}
+	logf(d1, d2) {
+		if (d1 <= 0.0) return NEGATIVE_INFINITY
+		if (d2 <= 0.0) return NEGATIVE_INFINITY
+		return lnGamma(d1/2) + lnGamma(d2/2) - lnGamma((d1 + d2)/2)
```

Comment:
Better to combine logf blocks with identical dependencies (i.e. the first two here)